crates/client/src/client.rs

```diff
@@ -473,7 +476,9 @@ impl FuelClient {
         use cynic::SubscriptionBuilder;
         let s = schema::tx::StatusChangeSubscription::build(TxIdArgs { id: id.parse()? });
 
+        tracing::debug!("subscribing");
```

Comment:
should we include the tx id here or is that covered by the instrumentation span?

Reply:
It's included in the span above